idaes/models/properties/modular_properties/state_definitions/tests/test_FPhx.py

```diff
@@ -137,7 +138,7 @@ def test_mole_frac(self, caplog):
 
 class Test1PhaseDefinedStateFalseNoBounds(object):
     # Test define_state method with no bounds and defined_State = False
-    @pytest.fixture(scope="class")
+    @pytest.fixture()
```

Comment:
Just out of curiosity, why are we removing these scopes?

Reply:
They resulted in tests polluting other tests which resulted in unexpected failures (mainly the first check for defined state effect had flow on effects to other tests)